pkg/controller/kieapp/constants/constants.go

```diff
@@ -12,3 +12,9 @@ const (
 	ImageStreamTag                    = "1.0"
 	ConfigMapPrefix                   = "kieconfigs"
 )
+
+// MonitoringEnvs Type of environments that will deploy the Monitoring console.
+// The console resources will be suffixed as -monitoring as well
+var MonitoringEnvs = map[string]struct{}{
+	"production": struct{}{},
```

Comment:
How about this instead?

var MonitoringEnvs = map[string]bool{
	"production": true,
}